compose.yml

```diff
@@ -95,8 +95,21 @@ services:
     depends_on:
       - keycloak
 
+  frontend:
+    user: "${DOCKER_USER:-1000}"
+    build:
+      context: .
+      dockerfile: ./src/frontend/Dockerfile
+      target: frontend-production
+      args:
+        VITE_API_BASE_URL: "http://localhost:8071"
+    image: meet:frontend-development
+    ports:
+      - "3000:8080"
+
   dockerize:
     image: jwilder/dockerize
+    platform: linux/x86_64
```

Comment:
dockerize can be fully remove. See this commit https://github.com/suitenumerique/docs/commit/0dc8b4556cfc4d9194d291fd5c7e69fdef72fa75